.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: java
+
+jdk:
+  - oraclejdk11
+
+cache:
+  directories:
+    - .autoconf
+    - $HOME/.m2
+
+install: true
+script:
+  - cd jaxb-ri
+  - mvn -U -C -Pstaging clean install
```

Comment:
I'd prefer using 'oss-release' profile instead of 'staging' to avoid broken javadocs and/or sources generation

Reply:
Wouldn't that break the build as soon as staging dependency is integrated? How about leaving there both of them?